### PR TITLE
Apply changes from 2.9.1 Release ticket

### PR DESCRIPTION
**Pull request by gabi18**

### CODE REVIEW

File: .github/patch_release_template.md [hunk 1]
```diff
@@ -22,6 +21,7 @@ __TODO__: [patch_release_template.md](https://github.com/owncloud/client/blob/ma
 ### Build
 
 * [ ] DEV: Tag and build [builds](https://confluence.owncloud.com/display/OG/Build+and+Tags#BuildandTags-Sprintbuild) for theme 'ownCloud' and 'testpilotcloud' (includes ChangeLog for the tag on https://github.com/owncloud/client/releases/)
```
Comment (jnweiger) [inline .github/patch_release_template.md:23]:
```suggestion
* [ ] DEV: Tag and build [builds](https://confluence.owncloud.com/display/OG/Build+and+Tags#BuildandTags-Sprintbuild) for theme 'ownCloud' and 'testpilotcloud' (includes ChangeLog for the tag on https://github.com/owncloud/client/releases/)
* [ ] QA: Ping documentation-internal: Changelog is ready. (open doc issues for already known doc-relevant items)
```

File: .github/patch_release_template.md [hunk 2]
```diff
@@ -22,6 +21,9 @@ __TODO__: [patch_release_template.md](https://github.com/owncloud/client/blob/ma
...
+* [ ] QA: Ping #documentation-internal: Changelog is ready. (open doc issues for already known doc-relevant items)
+* [ ] QA: Ping #documentation-internal: Changelog is ready. (open doc issues for already known doc-relevant items)
```
Comment (jnweiger) [inline .github/patch_release_template.md:25]:
```suggestion
```

File: .github/patch_release_template.md [hunk 3]
```diff
@@ -31,34 +32,39 @@ __TODO__: [patch_release_template.md](https://github.com/owncloud/client/blob/ma
...
 * [ ] DEV: [Smash box test](https://drone.owncloud.com/owncloud/smashbox-testing) Make sure tests run on latest version 
-* [ ] QA: [Smoke test](https://confluence.owncloud.com/display/OG/Manual+Tests#ManualTests-DEVSmokeTest)
-* [ ] QA: Run [automated tests](https://confluence.owncloud.com/display/OG/Automated+Tests)
-    * [ ] TODO:  add "xvfb-run owncloud-client" to RUN.sh ([Xvfb](https://en.wikipedia.org/wiki/Xvfb))
+* [ ] QA: Run [automated tests](https://confluence.owncloud.com/display/OG/Automated+Tests) (includes [Smoke test](https://confluence.owncloud.com/display/OG/Manual+Tests#ManualTests-DEVSmokeTest))
+   * [ ] All Linux platform install and gpg test ssh://git@gitea.owncloud.services:2222/client/linux-docker-install.git
+       * [ ] manually deploy a linux download repo, or use a download repo from https://download.owncloud.com/desktop/ownCloud/testing/
+       * [ ] in defs.sh edit `repo=` and update `platform_docker_images=`
+       * [ ] RUN.sh -> paste the log/test_YYYYMMDD_hhmm.log file into the client release ticket.
+       * [X] DONE: "xvfb-run" and AppImage install are added (05-xvfb-run.sh, 08-appimage.sh)
```
Comment (gabi18) [inline .github/patch_release_template.md:40]:
```suggestion
       * [ ] TO DO: extend 08-appimage.sh ("xvfb-run")
```

File: .github/patch_release_template.md [hunk 4]
```diff
@@ -31,34 +32,39 @@ __TODO__: [patch_release_template.md](https://github.com/owncloud/client/blob/ma
...
 * [ ] DEV: Publish the release in GitHub
+* [ ] QA: Check [documentation](https://confluence.owncloud.com/display/OG/Documentation)
 
 ### Communicate the Availability
 
 * [ ] Final [Marketing and Communication](https://confluence.owncloud.com/display/OG/Marketing+and+Communication)
    * [ ] Website links for final release (needed for the following posts)
    * [ ] QA: Central post https://central.owncloud.org/tags/c/news/desktop
-   * [ ] QA: Inform other [stakeholders](https://confluence.owncloud.com/display/OG/Marketing+and+Communication#MarketingandCommunicationInformStakeholdersaboutFinal)
+   * [ ] QA: Inform on ``#updates`` channel,
```
Comment (gabi18) [inline .github/patch_release_template.md:66]:
```suggestion
   * [ ] QA: Inform on ``#updates`` channel, so that marketing knows about the new release
```

**Review by jnweiger — APPROVED** [review]
Looks good to me now. 
Thanks!

**Comment by jnweiger** [comment]
We can ignore "continuous-integration/drone/pr — Build is failing " here.

**Review by michaelstingl — APPROVED** [review]
LGTM

**Pull request merged**